### DISCUSSION TITLE
Streamlink Twitch is causing audio to skip and have a higher pitch only on one stream.

### DISCUSSION THREAD

**Discussion by Zeond1987** [streamlink/streamlink#5537] [Q&A] [answered]
### Description

It happens only on one Twitch stream and other Twitch streams work fine. In one instance, I restarted my computer and the stream was working fine again. But it still happens with that one and only stream and I don't get why. I don't know how to fix it and could someone please help me? Thank you.

**Comment by bastimeyer** [comment, marked as answer]
Your description doesn't include any information at all. How are we supposed to help you if all you say is that you're having an audio problem? You didn't even tell us which channel it is.

So, what's your operating system and what's the used player? Are you filtering embedded ads ([`--twitch-disable-ads`](https://streamlink.github.io/cli.html#cmdoption-twitch-disable-ads))? Was it after an ad-transition or after a transition while filtering embedded ads? These are stream discontinuities, which result in broken and incoherent output data. Streamlink can't fix this.

VLC is known to cause issues, as you can read here:
https://streamlink.github.io/cli/plugins/twitch.html#embedded-ads

If you're using VLC, then switch to MPV:
https://mpv.io/

**Reply by Zeond1987** [reply]
The channel is "Timmac" and my OS is Windows 11. The player is VLC and I'm not filtering ads. It happens during the entire stream, before or after an ad-transition. I'm sorry for not giving you any information, I don't use these forums much. Is there anything else I can tell you to help you? I have a sub to his channel so that should stop ads on the website and I just got Twitch turbo so maybe that should help?

Edit: I turned on the filter embedded ads and now everything is good! Thank you for your help!.
